### ISSUE TITLE
Backdating prescription UI

### ISSUE THREAD

**Issue by mark-prins** [msupply-foundation/open-msupply#4864]
## Is your feature request related to a problem? Please describe 👀
A tweak to the UI please.
The date field is floating in the middle of the toolbar area, not vertically aligned with the other inputs:

![image](https://github.com/user-attachments/assets/3e514db8-86dc-4085-8dd7-d26db27f5140)

Can we adjust so that the inputs align, and bring the date input closer to the others

![image](https://github.com/user-attachments/assets/25f66cd9-5144-40a5-83c3-7c9e8acf9c14)

## Describe the solution you'd like 🎁
In addition to the UI move above, I'd also like the following changes:

I'd also like to default the value to today's date - leaving the input empty invites the user to enter something, which would be the wrong intention. Defaulting to today gives the user a clue that the date is the date of the prescription and therefore will be today's date by default.

Additional thoughts
* maybe move it to the 'more info' panel?
* if so, shift clinician up to make the toolbar smaller vertically
* rename? The `Date` label isn't telling me what the intention is. Perhaps if we move to the info panel we could add some help text?

**Comment by andreievg** [comment]
@mark-prins can we default the date to creation date ? Otherwise we can to default to today date, or picked date or allocated date (the latter is the one we use as backdate date). And the reason to also use picked date, is because otherwise todays date would be shown on historic prescriptions ?

**Comment by mark-prins** [comment]
yes - creation date is the intention: you're right we don't want to show today's date on historic prescriptions.

**Comment by jmbrunskill** [comment]
> @mark-prins can we default the date to creation date ? Otherwise we can to default to today date, or picked date or allocated date (the latter is the one we use as backdate date). And the reason to also use picked date, is because otherwise todays date would be shown on historic prescriptions ?

Nice, I like this, will jump on it now...

**Comment by roxy-dao** [comment]
Closed in #4877

**Comment by marthakeezy** [comment]
Tested in V2.3.0, moving to Done ✅
